npm-shrinkwrap.json

```diff
@@ -1,322 +0,0 @@
-{
```

Comment:
Oh, right - ha. Can you tell it's been a while since I thought about this module? ;)

Sounds good!